### PR TITLE
emissary-only manifests and helm chart

### PR DESCRIPTION
**Pull request by acookin**
## Description
Switch to using the emissary only chart for realz, get rid of ambassador chart and clean up aes manifests.

Probably best to review the commits.
The first two commits are me doing `git rm -rf charts/emissary-ingress` `git mv charts/ambassador charts/emissary-ingress` so those aren't interesting.
The 3rd commit should be the diff between the emissary and ambassador charts.
4th commit is handling the changes to flat manifest files
all other commits are mostly fixups.

There will be a PR coming for apro.git for the edge-stack only chart and yaml.

As far as "migrating" to this chart, if you had installed the old ambassador chart with something like:
```
helm install --set enableAES=false ambassador -n ambassador datawire/ambassador
```
You would be able to upgrade to the emissary chart with:
```
helm upgrade ambassador -n ambassador ./charts/emissary-ingress
```

## Related Issues
List related issues.

## Testing
A few sentences describing what testing you've done, e.g., manual tests, automated tests, deployed in production, etc.

## Checklist

 - [ ] I made sure to update `CHANGELOG.md`.
   
   Remember, the CHANGELOG needs to mention:
    + Any new features
    + Any changes to our included version of Envoy
    + Any non-backward-compatible changes
    + Any deprecations
 
 - [ ] This is unlikely to impact how Ambassador performs at scale.
 
   Remember, things that might have an impact at scale include:
    + Any significant changes in memory use that might require adjusting the memory limits
    + Any significant changes in CPU use that might require adjusting the CPU limits
    + Anything that might change how many replicas users should use
    + Changes that impact data-plane latency/scalability
 
 - [ ] My change is adequately tested.
 
   Remember when considering testing:
    + LEGACY MODE TESTS DO NOT RUN FOR EVERY PR. If your change is affected by legacy mode, you need
      to run legacy-mode tests manually (set `AMBASSADOR_LEGACY_MODE=true` and run the tests).
      (This will be fixed soon.)
    + Your change needs to be specifically covered by tests.
       + Tests need to cover all the states where your change is relevant: for example, if you add a behavior that can be enabled or disabled, you'll need tests that cover the enabled case and tests that cover the disabled case. It's not sufficient just to test with the behavior enabled.
    + You also need to make sure that the _entire area being changed_ has adequate test coverage.
       + If existing tests don't actually cover the entire area being changed, add tests.
       + This applies even for aspects of the area that you're not changing – check the test coverage, and improve it if needed!
    + We should lean on the bulk of code being covered by unit tests, but...
    + ... an end-to-end test should cover the integration points
 
 - [ ] I updated `DEVELOPING.md` with any any special dev tricks I had to use to work on this code efficiently.

### CODE REVIEW

File: python/ambassador_diag/diagd.py [hunk 1]
```diff
@@ -535,27 +535,8 @@ def check_cache(self) -> bool:
         return result
 
 
-# get the "templates" directory, or raise "FileNotFoundError" if not found
-def get_templates_dir():
```
Comment (acookin) [inline python/ambassador_diag/diagd.py:539]: this was.. not used?

File: charts/emissary-ingress/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,23 +1,21 @@
-# Contributing to the Ambassador Helm Chart
+# Contributing to the Emissary Ingress Helm Chart
 
-This Helm chart is used to install The Ambassador Edge Stack (AES) and is 
-maintained by Datawire.
+This Helm chart is used to install The Emissary Ingress.
```
Comment (kflynn) [inline charts/emissary-ingress/CONTRIBUTING.md]: `s/The/the/`

File: charts/emissary-ingress/templates/ambassador-agent.yaml [hunk 3]
```diff
@@ -5,20 +5,16 @@
   {{- /* Case 1: "fullImageOverride" means don't bother checking the tag. */ -}}
   {{- /* Case 2: Otherwise, if it's not a semver-style version number,    */ -}}
   {{- /*         assume we have a power user and turn the agent on.       */ -}}
-  {{- /* Case 3: Otherwise, if Edge Stack, we need at least 1.12.0.       */ -}}
-  {{- /* Case 4: Otherwise, it's OSS and we need at 1.13.0.               */ -}}
+  {{- /* Case 3: Otherwise, it's OSS and we need at 1.13.0.               */ -}}
```
Comment (kflynn) [inline charts/emissary-ingress/templates/ambassador-agent.yaml]: `s/at 1.130/at least 1.13.0` ... and yeah, pretty sure that one was originally my bug. 😛 😂

**Review by kflynn — APPROVED** [review]
Two wicked minor nits, approving anyway -- thanks!! 🙂

File: charts/emissary-ingress/templates/ambassador-agent.yaml (hunk 3 above)
Comment (acookin) [reply charts/emissary-ingress/templates/ambassador-agent.yaml]: hmm wonder if we can just get rid of this for hte emissary chart.

**Comment by acookin** [comment]
> Two wicked minor nits, approving anyway -- thanks!! 🙂

I have fixes for these staged for a PR after I merge this (just so i can start working on the EA stuff and stop worrying about stale branches) if that's okay

**Review by esmet — APPROVED** [review]

**Pull request merged**